gruntfile.js

```diff
@@ -7,21 +7,22 @@ var _ = require('lodash'),
   defaultAssets = require('./config/assets/default'),
   testAssets = require('./config/assets/test'),
   fs = require('fs'),
-  path = require('path');
+  path = require('path'),
+  childProcess = require('child_process');
 
-module.exports = function (grunt) {
+module.exports = function(grunt) {
   // Project Configuration
-  var LOGO = "";
+  var LOGO = '';
   if (process.env.ENVIRONMENT === 'MEM') {
     LOGO = 'modules/core/client/img/brand/mem-logo-inverted.png'; // EAO Logo
   } else {
     LOGO = 'modules/core/client/img/brand/eao-banner-img-lg.png'; // BC Logo
   }
-  var ENV = "";
+  var ENV = '';
   if (process.env.ENVIRONMENT) {
     ENV = process.env.ENVIRONMENT;
   } else {
-    ENV = "EAO";
+    ENV = 'EAO';
```

Comment:
So *mem-admin* has *EAO* as ENV?

Reply:
No, the change here is the double quotes to single quotes.